rules/aws_cloudtrail_rules/aws_software_discovery.yml

```diff
@@ -0,0 +1,81 @@
+AnalysisType: rule
+Description: A user is obtaining a list of security software, configurations, defensive tools, and sensors that are in AWS.
+DisplayName: AWS Software Discovery
+Enabled: false
+Filename: aws_software_discovery.py
+Reference: https://attack.mitre.org/techniques/T1518/001/
+Reports:
+    MITRE ATT&CK:
+        - T1518:T1518.001
```

Comment:
my understanding is that the web UI doesn't handle subtechniques. 
I think this will need to be `TAxxxx:Txxxx`